versions.lock

```diff
@@ -29,7 +29,7 @@ com.google.cloud:google-cloud-core:1.94.3 (3 constraints: da2307c0)
 com.google.cloud:google-cloud-core-http:1.94.3 (2 constraints: 651543d8)
 com.google.cloud:google-cloud-storage:1.113.14 (2 constraints: 10143aa3)
 com.google.code.findbugs:annotations:3.0.1 (1 constraints: 0605fb35)
-com.google.code.findbugs:jsr305:3.0.2 (9 constraints: 057f5545)
+com.google.code.findbugs:jsr305:3.0.1 (1 constraints: b70f3485)
```

Comment:
one remains...

Reply:
I think this is the testOnly inclusion mentioned earlier - the binary distro now does not include these jars. I think the test-only include is to support the annotations in IDEs